six/modules/c++/six.sicd/include/six/sicd/Utilities.h

```diff
@@ -329,6 +329,14 @@ class Utilities
             const std::vector<std::string>& schemaPaths =
                     std::vector<std::string>(),
             logging::Logger* logger = NULL);
+
+    /*!
+     * Create a fake SICD that's populated enough for
+     * general testing code to run without throwing exceptions
+     *
+     * \return mock ComplexData object
+     */
+    static ComplexData createFakeComplexData();
```

Comment:
This seems like a good idea.  The one thing is can you change it to return an `auto_ptr` to a `ComplexData` to avoid the copy?